loki.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     "^embed",
     "^design system",
     "^Inputs/DatePicker Dates range",
+    "^Buttons/Button Compact size, custom color",
```

Comment:
Fix the conflict here by removing the story filter [that hasn't been backported.](https://github.com/metabase/metabase/blob/fbf0e7a3ad35fdd78fc56554fa6600c1157f4fe5/loki.config.js#L10)